src/applications/claims-status/containers/ClaimStatusPage.jsx

```diff
@@ -75,6 +76,7 @@ class ClaimStatusPage extends React.Component {
         <ClaimStatusHeader claim={claim} />
         {isOpen ? (
           <>
+            {decisionLetterSent ? <OpenClaimDecisionLetterAlert /> : <></>}
```

Comment:
You can just do `{decisionLetterSent && <OpenClaimDecisionLetterAlert />}` here instead